src/GitUrlTagger.ts

```diff
@@ -51,6 +56,42 @@ export class GitUrlTagger implements IAspect {
     return ''; // .git directory not found
   }
 
+  findRootDirectory(): string {
+    let currentDir = process.cwd(); // Get the current directory
+
+
+    while (currentDir !== '/') {
+      const packageJson = path.join(currentDir, 'package.json');
+
+
+      if (fs.existsSync(packageJson)) {
+
+        return currentDir;
+      }
+
+      currentDir = path.dirname(currentDir); // Move up to the parent directory
+    }
+    return ''; // root directory not found
+  }
+
+  pullGitUrlFromFile() {
+    const rootpath = this.findRootDirectory();
+    const gitUrlTaggerConfig = path.join(rootpath, 'git-url-tagger.json');
```

Comment:
because this is a configuration file, let's put a `.` in front of it.

```suggestion
    const gitUrlTaggerConfig = path.join(rootpath, '.git-url-tagger.json');
```

Reply:
Change implemented